git/git.zsh

```diff
@@ -0,0 +1,282 @@
+# The git prompt's git commands are read-only and should not interfere with
```

Comment:
I'm happy to keep this for now, and let's keep an eye on 1) is it going to slowdown performance, and 2) if it's going to make it hard to customize prompts when using oh-my-zsh, others.

Reply:
👍